packages/patternfly-4/react-core/src/components/Button/Button.tsx

```diff
@@ -25,7 +25,7 @@ export interface ButtonProps extends React.HTMLProps<HTMLButtonElement> {
   /** Additional classes added to the button */
   className?: string;
   /** Sets the base component to render. defaults to button */
-  component?: React.ReactNode;
+  component?: React.ReactNode | React.ComponentType<any>;
```

Comment:
Does just `component?: React.Element` work? I _think_ it should and that it's the better type to use.

Reply:
I assume you meant `React.ElementType`.

Reply:
Yessir!